src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -1515,6 +1629,8 @@ final gift = gifts['first'];
 
 {{site.alert.note}}
   The index operator (`[]`) returns a **nullable** value.
+  The operator returns `null` if the map does not include
+  the lookup key.
```

Comment:
Saying it returns a nullable value and it returns `null` seems confusing and a bit unnecessary. Can we just say one or the other, or connect them better somehow?